lib/tool_belt/koji_tools.rb

```diff
@@ -31,11 +31,16 @@ def ensure_packages(tag_name, packages)
       koji_change("add-pkg #{tag_name} #{packages.join(' ')} --owner=kojiadm")
     end
 
-    def ensure_package_group(source_tag, dest_tag, package_group)
+    def ensure_package_group(dest_tag, package_group, options)
       unless has_package_group?(dest_tag, package_group)
         create_package_group(dest_tag, package_group)
       end
-      packages = packages_for_group(source_tag, package_group)
+      packages = if source_tag = options.fetch(:source_tag, false)
+                   packages_for_group(source_tag, package_group)
+                 else
+                   options.fetch(:packages)
```

Comment:
should `options.fetch(:packages)` also have a fallback default?

Reply:
My original idea was no, in that if you don't at least supply packages this should fail which fetch will. The alternative would be adding an empty set of packages which could be viable. @komidore64 got a preference?

Reply:
yeah that makes sense. i'm okay with it failing if no packages were given.